src/requirements/data/minors/cs.ts

```diff
@@ -25,12 +30,34 @@ const csMinorRequirements: readonly CollegeOrMajorRequirement[] = [
     fulfilledBy: 'courses',
     perSlotMinCount: [1],
   },
-  // TODO: Needs excludes for certain classes
+  // TODO: Temp. excluding courses for Req 2
   {
     name: '4 CS courses numbered 3000 or higher',
     description: 'CS 4090, CS 4997, CS 4998, CS 4999 and seminars are excluded. CS 2800 is allowed',
     source: 'https://www.cs.cornell.edu/undergrad/csminor',
-    checker: includesWithSingleRequirement('CS 2800', 'CS 3***', 'CS 4***', 'CS 5***', 'CS 6***'),
+    checker: [
+      (course: Course): boolean => {
+        if (
+          courseMatchesCodeOptions(course, [
+            'CS 4090',
+            'CS 4997',
+            'CS 4998',
+            'CS 4999',
+            'CS 7***',
+            'CS 3110',
```

Comment:
temporarily excluding courses that fulfill other requirements in the same minor to be extra safe as we don't have ways to disallow double counting for minors